style.css

```diff
@@ -109,3 +129,199 @@ p {
 .logo2 {
   padding: 20px 20px 20px 0;
 }
+
+.header2 {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  align-items: center;
+}
+
+.ha {
+  width: 327px;
+  height: 83px;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 35px;
+  line-height: 52px;
+  color: #1c1a19;
+  padding: 24px;
+}
+
+.vector {
+  width: 327px;
+  border: 1px solid #212121;
+}
+
+.cont {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.name {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  align-items: center;
+}
+
+.hb {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+}
+
+.hc {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.nx {
+  display: flex;
+  gap: 12px;
+  flex-direction: row;
+  list-style-type: none;
+  padding-left: 24px;
+}
+
+button {
```

Comment:
- [ ] I kindly suggest that you add the correct button interactions including hover and active (when it's pressed), as shown in the Figma template below:

<img width="541" alt="image" src="https://user-images.githubusercontent.com/52934128/193900851-3cb52cfc-3deb-4bfa-9ad8-054f39c6ea76.png">

the button on the pressed state has the `background: #E05304` and on hover has the `background: #FFF5E1`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #E05304;
}
```
**for hover state**
```
button:hover {
    background-color: #FFF5E1;
}
```